### DISCUSSION TITLE
Could not generate CGLIB subclass of class software.amazon.awssdk.services.sqs.DefaultSqsAsyncClient

### DISCUSSION THREAD

**Discussion by lebkset** [aws/aws-sdk-java#3047] [General]
I used Spring boot 3.1.2 , spring cloud 3.0.2 and aws.java.sdk 2.20.55 and use SQS and I run application fail with error :

`Error creating bean with name 'sqsAsyncClient' defined in class path resource [io/awspring/cloud/autoconfigure/sqs/SqsAutoConfiguration.class]: Could not generate CGLIB subclass of class software.amazon.awssdk.services.sqs.DefaultSqsAsyncClient: Common causes of this problem include using a final class or a non-visible `

**Comment by debora-ito** [comment]
Yes, `DefaultSqsAsyncClient` is final. This seems like a requirement on the CGLIB generation, not really an issue with the Java SDK. What's your ask for the AWS Java SDK team?
